config/config.yaml

```diff
@@ -89,9 +95,35 @@ MODEL_FOR_RESEARCHER_REPORT: gpt-3.5-turbo-16k
 
 ### choose the engine for mermaid conversion, 
 # default is nodejs, you can change it to playwright,pyppeteer or ink
-# MERMAID_ENGINE: nodejs
+#MERMAID_ENGINE: nodejs
+
 
 ### browser path for pyppeteer engine, support Chrome, Chromium,MS Edge
 #PYPPETEER_EXECUTABLE_PATH: "/usr/bin/google-chrome-stable"
 
-PROMPT_FORMAT: json #json or markdown
\ No newline at end of file
+PROMPT_FORMAT: json #json or markdown
+
+
+####Customizing the LLM used by Action and Role
+MULTI_LLM: True
+
+####Provide LLM
+####Currently support openai,claude,spark,zhipu,class openai
+###Customize the LLM used by the role
+ProductManager: llama2-13b-chat-hf
+Architect: llama2-13b-chat-hf
+ProjectManager: llama2-13b-chat-hf
+Engineer: llama2-13b-chat-hf
+#
+####Customizing the LLM used by Action
+WritePRD: llama2-13b-chat-hf
+WriteDesign: llama2-13b-chat-hf
+WriteTasks: llama2-13b-chat-hf
+WriteCode: openai
+WriteCodeReview: openai
+#
+#
+####Customize LLM name and request address
+MODEL_LIST:
+  llama2-13b-chat-hf: "http://0.0.0.0:17865/v1"
```

Comment:
the `MODEL_LIST` config seems to set the llm-infer-service config  

MODEL_LIST:
    llama2-13b-chat-hf:
        open_llm_api_base:  xxxx:7890
    llama2-13b-chat-hf:
        open_llm_api_base: xxxx:7891
    openai:
        openai_api_base: xxx
        ...